pages/en/deploying/subgraph-studio.mdx

```diff
@@ -64,7 +64,11 @@ If you’d like to test your subgraph before publishing it to the network, you c
 
 ## Publish your Subgraph in Subgraph Studio
 
-You’ve made it this far - congrats! Publishing your subgraph means that an IPFS hash was generated when you deployed the subgraph within the CLI and is stored in the network’s Ethereum smart contracts. In order to publish your subgraph successfully, you’ll need to go through the following steps outlined in this [blog](https://thegraph.com/blog/building-with-subgraph-studio). Check out the video overview below as well:
+You’ve made it this far - congrats!
+
+Publishing your subgraph means that an IPFS hash was generated when you deployed the subgraph within the CLI and is stored in the network’s smart contracts. In order to publish your subgraph successfully, you’ll need to go through the following steps outlined in this [blog](https://thegraph.com/blog/building-with-subgraph-studio).
```

Comment:
> Publishing your subgraph means that an IPFS hash was generated when you deployed the subgraph within the CLI and is stored in the network’s smart contracts.

I find this sentence confusing. Isn't that describing deploying, rather than publishing?

Reply:
I also found this one confusing. @Web3Slimchance could you clarify or make a suggestion here?

Reply:
@Web3Slimchance Good call - rewrote that!